test/presenters/content_item/contents_list_test.rb

```diff
@@ -84,12 +84,11 @@ def body
                  @contents_list.contents_items
   end
 
-  test "#show_contents_list? returns true if number of contents items is less than 3 and the first item's character count is above 415" do
+  test "#show_contents_list? returns true if the first item's character count is above 415" do
```

Comment:
This file needs to have the following test cases:

1. Displays contents list if there is an H2
```
def body
  "<h2 id='one'>One</h2>"
end
```
2. Displays contents list if there is no H2 but first item is greater than 415
```
def body
  "<div><p>#{Faker::Lorem.characters(number: 416)}</p></div>"
end
```

3. Displays no contents list if there is no H2 and first item is less than 415
```
def body
  "<div><p>#{Faker::Lorem.characters(number: 400)}</p></div>"
end
```
And then the tests for the other cases (images and tables)

I think all of them are present in this file, but you just need to tweak them so that the body provides the right structure. Hope that makes sense!